Makefile

```
@@ -108,6 +109,28 @@ tag: ## Set a docker tag to the image.
 push: ## Push docker image.
 	docker push $(IMAGE_PREFIX)pvc-autoresizer:$(IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: # Lint and validate a chart.
+	docker run \
+		--rm \
+		--workdir=/data \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct lint --config ct.yaml
```

```suggestion
	docker run \
	    --user $(id -u $USER)
		--rm \
		--workdir=/data \
		--volume $(shell pwd):/data \
		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
		ct lint --config ct.yaml
```

`ct lint --config ct.yaml` says error
`Error: failed linting charts: failed identifying charts to process: must be in a git repository` in my environment.
This is due to permission issues within the host and container.
Matching the user id in the host and container will solve the problem.

---

@pluser Thank you. I added `--user` option